@types/interfaces.d.ts

```diff
@@ -0,0 +1,13 @@
+export interface LoginI18n {
```

Comment:
This interface is missing the optional `header` and `additionalInformation` which are documented in JSDoc.

Reply:
Thanks, added.